src/main/resources/application.properties

```diff
@@ -17,9 +17,11 @@ AWS_ROLE_ARN=if_you_see_this_please_use_AWS_Setup
 AWS_WEB_IDENTITY_TOKEN_FILE=if_you_see_this_please_use_AWS_Setup
 FILENAME_CHALLENGE9=wrongsecret
 FILENAME_CHALLENGE10=wrongsecret-2
-azure.keyvault.enabled=false
-management.health.azure-key-vault.enabled=false
-azure.keyvault.uri=https://default.vault.localhost
+spring.cloud.azure.keyvault.secret.credential.client-id=false
+spring.cloud.azure.credential.managed-identity-enabled=false
+spring.cloud.azure.keyvault.secret.endpoint=https://default.vault.localhost
+spring.cloud.azure.keyvault.secret.profile.tenant-id=false4
+management.health.azure-keyvault-secret.enabled=false
```

Comment:
You could refer to https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#configuration-properties to see how to use Key Vault secret management.

Reply:
Changed it, however: the SDK does not seem to give back what is going wrong and why... So I don't understand what might now still be the problem.

Reply:
Thanks to your latest comment, it now works! Thank you!